src/Model/UserMeta.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Kirby\Model;
+
+class UserMeta extends ModelMeta
+{
+	public function __construct(
+		public string $identifier,
+		public string|null $email = null,
+		int $created = 0,
+		public string|null $language = null,
+		int $modified = 0,
+		public string|null $name = null,
+		public string|null $password = null,
```

Comment:
`Todo`: We should go for `$secrets` here as the `.htpasswd` file can also contain a TOTP secret and later other secrets (like the WebAuthn key). The `password` methods are only still present in `File` for compatibility. We shouldn't use this term going forward.